whatshap/threading.py

```diff
@@ -48,7 +45,12 @@ def run_threading(readset, clustering, ploidy, genotypes, block_cut_sensitivity)
     # we can look at the sequences again to use the most likely continuation, when a haplotype leaves a collapsed cluster (currently inactive)
     path = improve_path_on_collapsedswitches(path, num_clusters, c_to_c_global)
 
+    # determine cut positions
```

Comment:
That comment is redundant (the function name `compute_cut_positions` already says the same thing).